themes/openy_themes/openy_lily/sass/modules/_alerts.scss

```diff
@@ -46,6 +46,8 @@ $module: 'site-alert';
       text-align: center;
       &:hover {
         text-decoration: underline;
+        background-color: $white;
+        color: $black !important;
       }
```

Comment:
Let's use `opacity` on hover instead of hardcoded colors:

```css
opacity: 0.6;
```

![Screenshot from 2021-08-03 18-36-46](https://user-images.githubusercontent.com/13733670/128044808-cef6391a-06e5-45fb-b76e-c52bab3a46e8.png)
![Screenshot from 2021-08-03 18-36-53](https://user-images.githubusercontent.com/13733670/128044810-5c2ba39c-fa76-40cd-ac1b-0b45d4729ee6.png)


@podarok what do you think?

Reply:
@hamrant thank you for your suggestion.
I've made this fix in a new commit.
Please check it
cc @podarok